analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -899,9 +899,9 @@ public void Invocation_Equals_LearnResult(string left, string right, bool? expec
 Tag(""Left"", left);
 Tag(""Right"", right);";
             var validator = SETestContext.CreateCS(code).Validator;
-            validator.ValidateTag("Result", x => x.HasConstraint(BoolConstraint.From(expectedResult.Value)).Should().BeTrue());
-            validator.ValidateTag("Left", x => x.AllConstraints.Select(x => x.ToString()).OrderBy(x => x).JoinStr(", ").Should().Be(expectedConstraintsLeft));
-            validator.ValidateTag("Right", x => x.AllConstraints.Select(x => x.ToString()).OrderBy(x => x).JoinStr(", ").Should().Be(expectedConstraintsRight));
+            validator.ValidateTag("Result", x => x.HasConstraint(BoolConstraint.From(expectedResult)).Should().BeTrue());
+            validator.ValidateTag("Left", x => x.AllConstraints.Select(x => x.Kind).Should().BeEquivalentTo(expectedConstraintsLeft));
```

Comment:
I cross-checked this change by changing some expectations and the tests failed as they should.